plugins/guests/linux/cap/network_interfaces.rb

```diff
@@ -2,6 +2,11 @@ module VagrantPlugins
   module GuestLinux
     module Cap
       class NetworkInterfaces
+        # Valid ethernet device prefix values.
+        # eth - classic prefix
+        # en  - predictable interface names prefix
+        ETHERNET_PREFIX = ["eth", "en"]
```

Comment:
And possibly name it something more descriptive like `POSSIBLE_ETHERNET_PREFIXES`